index.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```

Comment:
- [ ] KIndly make sure the `test.md` file is ignored by git and is not present in your GitHub repository in order to meet all project's requirements :wink: 
- [ ] Kindly make sure to rename it `test.md` in order to meet the requirements :wink:

Reply:
@levy002 I have made changes